codec-http/src/main/java/io/netty/handler/codec/http/websocketx/WebSocketUtil.java

```diff
@@ -91,13 +89,9 @@ protected MessageDigest initialValue() throws Exception {
      * @return An encoded string containing the data
      */
     static String base64(byte[] data) {
-        ByteBuf encodedData = Unpooled.wrappedBuffer(data);
-        ByteBuf encoded = Base64.encode(encodedData);
-        String encodedString = encoded.toString(CharsetUtil.UTF_8);
-        encoded.release();
-        return encodedString;
+        byte[] encodedData = Base64.getEncoder().encode(data);
```

Comment:
@doom369 just wondering if we should call `encodeToString ` directly. WDYT ?

Reply:
@normanmaurer JavaDoc says that users should prefer explicit encoding as `encodeToString` is left is only for back compatibility.

Reply:
Hmm, I would expect that you would have to pick either url base64 or standard.   Which one does this use?

Reply:
@carl-mastrangelo that's actually a very good question... @doom369 can you verify that the output is what we expect ?

Reply:
@carl-mastrangelo default is standard - `Encoder.RFC4648` as it was in netty Base64.

`getUrlEncoder()` is `RFC4648_URLSAFE`
`getMimeEncoder()` is `RFC2045`

Reply:
> @normanmaurer JavaDoc says that users should prefer explicit encoding. As String constructor that is used inside `encodeToString` is deprecated.
> Also, inside `StandardCharsets.ISO_8859_1` is used. I think it doesn't matter for this particular case, but need to read spec first :).

@doom369 It makes no difference since base64 only uses subset of ASCII .. i.e. resulting String will be the same whether the encoding is ASCII, UTF_8, ISO_8859_1. The fact a deprecated method is used inside the JDK's impl I don't think is relevant, in fact it should be faster than explicit re-encoding since it "cheats". So I would recommend using `encodeToString` after all...

Reply:
@njhill thanks. Fixed.